### ISSUE TITLE
Customizable actions

### ISSUE THREAD

**Issue by mochigithub** [adirh3/Fluent-Search#21]
Customizable actions in FS's command menu. For example, Open, Open Directory, etc… These commands if customized could be very powerful.

Listary has implemented this pretty well.

![image](https://user-images.githubusercontent.com/1836362/120909305-abe78380-c628-11eb-9c56-0c1f7cc89776.png)

**Comment by Nomad-0** [comment]
![image](https://user-images.githubusercontent.com/76395594/120911149-8bf6a500-c6a6-11eb-9a2f-7e2a15429098.png)
Isn't it already implemented?

**Comment by mochigithub** [comment]
> Isn't it already implemented?
> 
> ![image](https://user-images.githubusercontent.com/76395594/120911149-8bf6a500-c6a6-11eb-9a2f-7e2a15429098.png)


No.  Your screenshot shows that the shortcuts can be configured but user cannot add custom actions like Listary. That's the request.
![image](https://user-images.githubusercontent.com/1836362/120914095-a2721180-c650-11eb-9528-eb2cb4593266.png)

**Comment by Metroid-Prime** [comment]
This is still one of my top requests after v1.0 is released. I am modifying the original request with a better picture and description

![image](https://user-images.githubusercontent.com/1836362/149642100-c6eb2a25-e204-4d2b-9349-44aa6825dd8f.png)

**Comment by dalmo3** [comment]
+1 for custom actions. That's probably my main use of Listary right now.

For example I have this custom action to open projects in vscode:
![image](https://user-images.githubusercontent.com/33860060/163706556-78dae6ee-cd65-4b47-a2f1-77cdfeb9811f.png)

The above is a very basic "Open With" and could be solved by https://github.com/adirh3/Fluent-Search/issues/552.

However most of the time I'm actually using WSL, so this is how the action looks like:
![image](https://user-images.githubusercontent.com/33860060/163707264-a7c3aecd-ad4d-42da-b8ea-7eb33e4f7a1b.png)

**Comment by Metroid-Prime** [comment]
Can't wait to see this @adirh3  The power of customizing these actions is going to be other-worldly.

**Comment by rzippo** [comment]
@dalmo3 With the tasks feature I was able to solve this for me, you should be able to import and adapt it with you command line arguments (remove the extension `.txt` first, GitHub did not let me upload as `.yaml`).

[VsCode.yaml.txt](https://github.com/adirh3/Fluent-Search/files/8541202/VsCode.yaml.txt)

**Comment by dalmo3** [comment]
You're right! I also had done it with Adir's help here https://github.com/adirh3/Fluent-Search/issues/552#issuecomment-1100884601
